### PR TITLE
Lynx: Add style variations.

### PR DESCRIPTION
**Pull request by alaczek**
This PR adds a few color style variations to the Lynx theme, along with a basic link in bio pattern.

### CODE REVIEW

File: lynx/styles/blue.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+				}
+			]
+		}
+	},
+	"styles": {
+		"color": {
+			"background": "var(--wp--preset--color--background)",
+			"text": "var(--wp--preset--color--foreground)"
+		}
+	}
```
Comment (mikachan) [inline lynx/styles/blue.json]:
```suggestion
	}
```

These are the same settings as the base theme, so you shouldn't need to redefine them in the style variation.

File: lynx/theme.json [hunk 2]
```diff
@@ -195,45 +198,51 @@
...
 		},
 		"color": {
-			"background": "#fff",
+			"background": "var(--wp--preset--color--background)",
 			"text": "var(--wp--preset--color--foreground)"
 		},
 		"elements": {
 			"h1": {
 				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--system-font)",
```
Comment (mikachan) [inline lynx/theme.json]:
```suggestion
```

Small nitpick, but as you're already setting the same default font family at `styles.typography.fontFamily`, you shouldn't need to set it again for each heading element.

File: lynx/theme.json [hunk 3]
```diff
@@ -195,45 +198,51 @@
...
 					"lineHeight": "var(--wp--custom--typography--line-height--small)",
 					"fontSize": "var(--wp--preset--font-size--xx-large)"
 				}
 			},
 			"h2": {
 				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--system-font)",
```
Comment (mikachan) [inline lynx/theme.json]:
```suggestion
```

File: lynx/theme.json [hunk 4]
```diff
@@ -195,45 +198,51 @@
...
 					"lineHeight": "var(--wp--custom--typography--line-height--small)",
 					"fontSize": "var(--wp--preset--font-size--x-large)"
 				}
 			},
 			"h3": {
 				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--system-font)",
```
Comment (mikachan) [inline lynx/theme.json]:
```suggestion
```

File: lynx/theme.json [hunk 5]
```diff
@@ -195,45 +198,51 @@
...
 					"lineHeight": "var(--wp--custom--typography--line-height--small)",
 					"fontSize": "var(--wp--preset--font-size--large)",
 					"fontStyle": "italic"
 				}
 			},
 			"h4": {
 				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--system-font)",
```
Comment (mikachan) [inline lynx/theme.json]:
```suggestion
```

File: lynx/theme.json [hunk 6]
```diff
@@ -195,45 +198,51 @@
...
 					"lineHeight": "var(--wp--custom--typography--line-height--medium)",
 					"fontSize": "var(--wp--preset--font-size--medium)",
 					"fontStyle": "italic"
 				}
 			},
 			"h5": {
 				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--system-font)",
```
Comment (mikachan) [inline lynx/theme.json]:
```suggestion
```

File: lynx/theme.json [hunk 7]
```diff
@@ -195,45 +198,51 @@
...
 					"textTransform": "uppercase",
 					"lineHeight": "var(--wp--custom--typography--line-height--normal)",
 					"fontSize": "var(--wp--preset--font-size--medium)"
 				}
 			},
 			"h6": {
 				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--system-font)",
```
Comment (mikachan) [inline lynx/theme.json]:
```suggestion
```

**Review by mikachan — COMMENTED** [review]
These are looking good! I really like the Neon variation. I've left a few minor comments.

File: lynx/styles/blue.json (hunk 1 above)
Comment (alaczek) [reply lynx/styles/blue.json]: Nicely spotted!

File: lynx/theme.json (hunk 2 above)
Comment (alaczek) [reply lynx/theme.json]: Yes, you're absolutely right! Fixed!

File: lynx/theme.json (hunk 4 above)
Comment (alaczek) [reply lynx/theme.json]: Fixed!

File: lynx/theme.json (hunk 5 above)
Comment (alaczek) [reply lynx/theme.json]: Fixed!

**Comment by simison** [comment]
Something is off with some texts still:

https://user-images.githubusercontent.com/87168/193035134-878ced88-178f-4c3c-b9f3-11749111616c.mov

Screenshot 2022-09-29 at 15 46 16

**Comment by alaczek** [comment]
Thanks for reporting @simison I'll take a look and open a new PR.

**Comment by simison** [comment]
Follow ups: https://github.com/Automattic/wp-calypso/issues/68465

**Comment by simison** [comment]
Some of the bugs in above videos fixed in https://github.com/Automattic/wp-calypso/issues/68512

**Pull request merged**